tests/core/support/jsdom-plugin.ts

```diff
@@ -67,6 +67,7 @@ intern.registerPlugin('jsdom', async () => {
 			}
 		});
 	};
+	initialize();
```

Comment:
This is because jsdom needs to be initialised for when the IO polyfill is required in order to run the tests in node. I think we should simply always initialise jsdom for testing rather than requiring a customised "describe" that implicitly performs an initialisation for tests it contains, but that should probably be a separate change.